src/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -184,6 +184,7 @@ private ICommand GetRunCommand()
         {
             var globalProperties = new Dictionary<string, string>
             {
+                { Constants.EnableDefaultItems,    "false" },
```

Comment:
Add a comment about why this property is set (perf), and why it's safe (we're only evaluating properties, not evaluating items or running targets, so it's not possible that item evaluation would affect the property values).

Reply:
I'll add a comment explaining the property use.